ui_framework/doc_site/src/views/menu/menu_example.js

```diff
@@ -1,39 +1,57 @@
 import React from 'react';
 
+import { renderToHtml } from '../../services';
+
 import {
   GuideDemo,
   GuidePage,
   GuideSection,
   GuideSectionTypes,
 } from '../../components';
 
-const menuHtml = require('./menu.html');
-const menuContainedHtml = require('./menu_contained.html');
+import Menu from './menu';
+const menuSource = require('!!raw!./menu');
```

Comment:
Interesting point... let's discuss this separately since it's how we're doing our examples elsewhere, so falls outside the scope of this PR.